### PR TITLE
Direction: rework reverse button

### PR DESCRIPTION
**Pull request by G-Ray**
## Screenshots
image

### CODE REVIEW

File: src/scss/includes/direction-form.scss [hunk 1]
```diff
@@ -32,19 +32,23 @@
 
     .direction-form-separator {
       border-top: 1px solid #c8cbd3;
-      margin: 0 37px;
+      margin: 0 52px 0 44px;
```
Comment (G-Ray) [inline src/scss/includes/direction-form.scss:35]: we might try to create a flex layout for the form elements later

Comment (bbecquet) [reply src/scss/includes/direction-form.scss:35]: Isn't it possible right now?

Comment (G-Ray) [reply src/scss/includes/direction-form.scss:35]: I would like to push this change and the font changes before, just to have the general visual elements in place before changing layout/DOM :)

Comment (G-Ray) [reply src/scss/includes/direction-form.scss:35]: I just added minor font changes

File: src/panel/direction/DirectionForm.jsx [hunk 2]
```diff
@@ -84,11 +84,12 @@ export default class DirectionForm extends React.Component {
           onChangePoint={(input, point) => this.onChangePoint('destination', input, point)}
           ref={this.destinationRef}
         />
-        <div
-          className="direction-invert-origin-destination icon-reverse"
+        <button
```
Comment (bbecquet) [inline src/panel/direction/DirectionForm.jsx:87]: Great to use a real button, but in this case you need to add a `type="button"` attribute, as the default type of a HTML button is `submit`, which will try to submit the form it belongs too… which will open a new page :D

**Review by bbecquet — DISMISSED** [review]

**Review by bbecquet — APPROVED** [review]

**Pull request merged**